app/src/main/java/com/github/emilehreich/bootcamp/GreetingActivity.kt

```diff
@@ -1,18 +1,22 @@
 package com.github.emilehreich.bootcamp
-
+import android.annotation.SuppressLint
+import android.content.Intent
 import android.os.Bundle
+import android.widget.EditText
 import android.widget.TextView
+import androidx.activity.compose.setContent
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material3.Text
+
 
 class GreetingActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_greeting)
-
-        val greet = findViewById<TextView>(R.id.greetingName)
         val name = intent.getStringExtra("name")
+        setContent{
+            Text(text = "Welcome $name")
+        }
```

Comment:
How do you make sure the created text is centered/how do you set its position? Is it by default at the center of the activity?

Reply:
In fact I did not. I will add it in the next commit